.github/workflows/test.yml

```
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        version: [1.0.0, 1.5.4, latest]
+        version: [latest]
```

```suggestion
        version: [1.0.0, latest]
```

let's still keep 1.0.0 for now since it's the default version

---

should we change the default to `latest`? I don't see what's special about `1.0.0`